core/modules/cart/store/actions/totalsActions.ts

```diff
@@ -43,7 +43,7 @@ const totalsActions = {
 
     if (getters.canSyncTotals && (getters.isTotalsSyncRequired || payload.forceServerSync)) {
       const shippingMethodsData = methodsData || createOrderData({
-        shippingDetails: rootGetters['checkout/getShippingDetails'],
```

Comment:
I’d suggest “getShippingDetails” as it will be more self explaining when used in the current context (in the shipping module where you call just “getShippingDetails” without the module name); the same note for other getters